control-station/src/services/usePodData.tsx

```diff
@@ -3,8 +3,14 @@ import PodSocketClient, { PodData, State } from "./PodSocketClient";
 
 function usePodData() {
 	const [podData, setPodData] = useState<PodData>({
-		state: State.Disconnected,
 		connected: false,
```

Comment:
I think we can remove this line because the state will already tell us whether the GUI is connected to the pod or not.